bootstrap.py

```diff
@@ -401,8 +402,8 @@ def install_puppet_agent():
     print_generic("if auto-signing is disabled")
     exec_failexit("/usr/bin/puppet agent --test --noop --tags no_such_tag --waitforcert 10")
     if 'puppet-enable' not in options.skip:
-        exec_failexit("/sbin/chkconfig puppet on")
-        exec_failexit("/sbin/service puppet restart")
+        enable_service("puppet")
+        start_service("restart", "puppet")
```

Comment:
It can't be cause the command parameter has assigned a default value equal to start at definition time. That's why the service which has no default value at definition time must be prior to command otherwise you should expect a python error.